packages/api/src/types.ts

```diff
@@ -35,6 +35,7 @@ export enum Network {
   HarmonyTestnet = 'harmony-testnet',
   MetisMainnet = 'metis-mainnet',
   MetisRinkeby = 'metis-rinkeby',
+  MoonbeamMoonbase = 'moonbeam-moonbase',
```

Comment:
for now it is the only way to go, but as a general comment to be considered as an upcoming issue:
- having to add an enum value everytime there's a new network addeded to the config file, totally breaks the more convenient possibility of not having to change code at all everytime a new network is added

Reply:
True and the perfect scenario is share a config file between machines. However, most probably we'll have to deploy every time we deploy a new network because we'll have to add a new image

Reply:
Yep, but images/logos could certainly be taken from a static repository, if we add this url as an additional parameter within the config file